modules/navier_stokes/src/fvbcs/WCNSFVEnergyFluxBC.C

```diff
@@ -25,8 +25,8 @@ WCNSFVEnergyFluxBC::validParams()
   // 3) Postprocessors for mass flow rate and energy, functor for specific heat
   params.addParam<PostprocessorName>("energy_pp", "Postprocessor with the inlet energy flow rate");
   params.addParam<PostprocessorName>("temperature_pp", "Postprocessor with the inlet temperature");
-  params.addParam<MooseFunctorName>(NS::cp, "specific heat capacity functor");
-
+  params.addRequiredParam<MooseFunctorName>(NS::cp, "specific heat capacity functor");
+  params.addRequiredParam<MooseFunctorName>("temperature", "temperature functor");
```

Comment:
I suggest using

```suggestion
  params.addRequiredParam<MooseFunctorName>(NS::T_fluid, "temperature functor");
```

or 

```suggestion
  params.addRequiredParam<MooseFunctorName>(NS::temperature, "temperature functor");
```